static/app/components/profiling/flamegraph/flamegraph.tsx

```diff
@@ -180,6 +180,10 @@ function Flamegraph(props: FlamegraphProps): ReactElement {
           minWidth: flamegraph.profile.minFrameDuration,
           barHeight: flamegraphTheme.SIZES.BAR_HEIGHT,
           depthOffset: flamegraphTheme.SIZES.FLAMEGRAPH_DEPTH_OFFSET,
+          configSpaceTransform:
+            xAxis === 'transaction'
+              ? new Rect(flamegraph.profile.startedAt * 1e3, 0, 0, 0)
```

Comment:
Why `1e3`? Is this to do with some unit conversion? If so, should this check the actual unit?

Reply:
@Zylphrex yeah, I hardcoded this and forgot to get back to it. I'm not sure if backend already returns profile.startedAt expressed profile.unit, but I assume it should so we dont need to do anything here, can you double check that?

Reply:
The `startedAt` value is expressed in the `unit` already. And as far as I can tell, we only use nanoseconds.